### PR TITLE
OZ-593: Updated E2E test for odoo-openmrs-flows

### PR DESCRIPTION
**Pull request by VaishSiddharth**
JIRA link: https://mekomsolutions.atlassian.net/browse/OZ-593

### CODE REVIEW

**Comment by rbuisson** [comment]
This should only be merged after [OZ-545: eip-odoo-openmrs to use openmrs-fhir](https://mekomsolutions.atlassian.net/browse/OZ-545) is done.

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ node_modules/
 /test-results/
 /playwright-report/
 /playwright/.cache/
+.idea/
```
Comment (kdaud) [inline .gitignore:5]: @VaishSiddharth is this change needed?

Comment (VaishSiddharth) [reply .gitignore:5]: Yes I added this otherwise `.idea` folder might get pushed to github.
`.idea` folder is created by IDE (IntelliJ IDEA)

**Comment by kdaud** [comment]
Note: This PR will wait to be merged until the changes from [OZ-545](https://mekomsolutions.atlassian.net/browse/OZ-545) are propagated on the Dev server. After that, E2E tests will be triggered to run against the changes in this commit.
